### PR TITLE
feat: ✨adds Change Requests reviews at card level

### PR DESCRIPTION
**Pull request by fpigeonjr**
## What changed

- 🟢 adds Change Requests(CR) reviews at card level
- content updates in Alerts and Modals
- updates e2e cypress tests for Change Requests 

## Issue

#2348 #2487

## How to test

1. Login as admin.
2. Add a budget or status change to Agreement 9.
3. Goto review page.
4. Approve and decline review cards.
5. 💰 profit

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/d38a65d1-4973-4fda-a4b4-5c39a714c2f2

## Definition of Done Checklist
- [X] OESA: Code refactored for clarity
- [X] OESA: Dependency rules followed
- [X] Automated unit tests updated and passed
- [X] Automated integration tests updated and passed
- [x] Automated quality tests updated and passed
- [X] Automated load tests updated and passed
- [x] Automated a11y tests updated and passed
- [X] Automated security tests updated and passed
- [X] 90%+ Code coverage achieved
- [-] Form validations updated

### CODE REVIEW

**Review by johndeange — APPROVED** [review]
It would be better to include the additional tests in the PR (the PR description says the tests are to come later) but I trust that you will remember to add them however, you may want to uncheck some of the DOD check boxes, e.g. "Automated integration tests updated and passed"

**Review by rajohnson90 — APPROVED** [review]

**Comment by fpigeonjr** [comment]
> It would be better to include the additional tests in the PR 

Thanks @johndeange, I'll add them in before merging

File: backend/openapi.yml [hunk 1]
```diff
@@ -1654,7 +1654,7 @@ paths:
                   type: integer
                 action:
                   type: string
-                  description: 'APROVE or REJECT'
+                  description: 'APPROVE or REJECT'
```
Comment (jonnalley) [inline backend/openapi.yml:1657]: @fpigeonjr are you implying we need to spell words correctly in our code ?

File: backend/openapi.yml [hunk 2]
```diff
@@ -1185,7 +1185,7 @@ paths:
             schema:
               $ref: '#/components/schemas/ServicesComponentRequestPost'
       responses:
-        '201':
```
Comment (fpigeonjr) [inline backend/openapi.yml:1188]: bring back to `201`

**Pull request merged**